soc/espressif/common/Kconfig.wifi

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_FAMILY_ESPRESSIF_ESP32
+
+menu "Wi-Fi config"
+
+config ESP_WIFI_MAX_THREAD_PRIORITY
```

Comment:
I guess it's too late but it would be nicer to have `ESP32` here as well.

Reply:
you got it!

Reply:
I meant the `ESP_WIFI_MAX_THREAD_PRIORITY` option, but it requires another change in esp32 hal repo, so let's keep it.